src/infrastructure/AccountHttp.ts

```diff
@@ -104,8 +147,31 @@ export class AccountHttp extends Http implements AccountRepository {
         const accountIds = {
             addresses: addresses.map((address) => address.plain()),
         };
-        return observableFrom(
-            this.accountRoutesApi.getAccountPropertiesFromAccounts(accountIds)).pipe(map((accountProperties) => {
+        const postBody = accountIds;
```

Comment:
:warning: not correct.

you should be copying over the nem2-library/src/api/* classes into the SDK because they are swagger generated.